statsmodels/sandbox/stats/multicomp.py

```diff
@@ -599,6 +618,7 @@ class TukeyHSDResults(object):
     std_pairs : standard deviation of pairwise mean differences
     q_crit : critical value of studentized range statistic at given alpha
     halfwidths : half widths of simultaneous confidence interval
+    p_adjs : adjusted p-values
```

Comment:
name: `p_adjs` -> `pvalues`
pvalues should be spelled out instead of just one letter `p`.
I think, given that the function or class has already `TukeyHSD` in the name, we don't need to add `_adjusted` to the name.